inst/examples/build-a-box/app.R

```diff
@@ -0,0 +1,242 @@
+# Packages ---------------------------------------
+library(shiny)
+library(bslib)
+
+pkgs_extra <- c("plotly", "colourpicker")
+pkgs_yes <- vapply(pkgs_extra, rlang::is_installed, logical(1))
+if (any(!pkgs_yes)) {
+  rlang::abort(paste0(
+    "The `build-a-box` app requires additional packages: ",
+    paste(pkgs_extra[!pkgs_yes], collapse = ", ")
+  ))
+}
+
+library(plotly)
+
+# Settings ---------------------------------------
+ENABLE_THEMER <- identical(Sys.getenv("ENABLE_THEMER"), "true")
+
+# Functions ---------------------------------------
+layout_value_box_options <- function(ui_opts) {
+  layout_columns(
+    div(h4("Content", class = "border-bottom"), ui_opts$title_value),
+    div(h4("Theme", class = "border-bottom"), ui_opts$theme_opts),
+    div(h4("Showcase", class = "border-bottom"), ui_opts$showcase)
+  )
+}
+
+value_box_placeholder <- function(id) {
+  value_box(
+    id = id,
+    class = "placeholder-glow",
+    title = span(class = "placeholder col-7"),
+    value = span(class = "placeholder col-4"),
+    showcase = div(class = "placeholder bg-primary col-12", as_fill_item())
+  )
+}
+
+# Theme ---------------------------------------
+theme_build_a_box <- bs_add_rules(
+  bs_theme(preset = "shiny"),
+  sass::sass_file("www/build-a-box.scss")
+)
+
+# UI ---------------------------------------
+ui <- page_fixed(
+  title = "Build a Box | bslib",
+  theme = theme_build_a_box,
+
+  # Header ----
+  tags$header(
+    h2(
+      class = "mt-4",
+      # width = "100%",
+      "Build a Box",
+      popover(
+        bsicons::bs_icon(
+          "info-square-fill",
+          title = "About Value Boxes",
+          class = "float-end icon-gradient"
+        ),
+        title = "About Value Boxes",
+        HTML(commonmark::markdown_html(readLines("about-value-boxes.md")))
+      )
+    ),
+  ),
+
+  # Main ----
+  tags$main(
+    # Value Box Previews ----
+    div(
+      id = "preview",
+      class = "my-5",
+      layout_columns(
+        class = "value-box-previews",
+        div(
+          as_fill_carrier(),
+          value_box_placeholder("one-value_box_placeholder"),
+          ui_value_box_output("one")
+        ),
+        div(
+          as_fill_carrier(),
+          value_box_placeholder("two-value_box_placeholder"),
+          ui_value_box_output("two")
+        ),
+        div(
+          as_fill_carrier(),
+          value_box_placeholder("three-value_box_placeholder"),
+          ui_value_box_output("three")
+        )
+      )
+    ),
+
+    # Settings Panel ----
+    navset_card_pill(
+      id = "settings",
+      title = span(
+        "Value box settings",
+        popover(
+          bsicons::bs_icon("question-square-fill"),
```

Comment:
I don't feel super strongly about this, but I thikn it'll work better in dark mode

```suggestion
          bsicons::bs_icon("question-square"),
```

Reply:
I'm going to stick with the filled version, thanks!